ui/hooks/gasFeeInput/test-utils.js

```diff
@@ -122,6 +126,12 @@ export const generateUseSelectorRouter = ({
       balance: '0x440aa47cc2556',
     };
   }
+  if (selector === getCustomMaxFeePerGas) {
+    return '0x5208';
+  }
+  if (selector === getCustomMaxPriorityFeePerGas) {
+    return '0x5208';
+  }
```

Comment:
[nit]
```suggestion
  if (selector === getCustomMaxFeePerGas || selector === getCustomMaxPriorityFeePerGas) {
    return '0x5208';
  }
```

Reply:
This condition is not logically same, just that I am returning same value for tests